yarn/src/main/scala/org/apache/spark/deploy/yarn/ExecutorRunnable.scala

```diff
@@ -128,6 +128,11 @@ private[yarn] class ExecutorRunnable(
     }
   }
 
+  // Kill if OOM is raised - leverage yarn's failure handling to cause rescheduling.
```

Comment:
This should be a comment on `addOutOfMemoryErrorArgument` right? that's what I meant.